pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -434,6 +448,70 @@ func GatherClusterOperators(i *Gatherer) func() ([]record.Record, []error) {
 	}
 }
 
+func collectClusterOperatorResources(i *Gatherer, co configv1.ClusterOperator, resVer map[string][]string) []clusterOperatorResource {
+	var relObj []configv1.ObjectReference
+	for _, ro := range co.Status.RelatedObjects {
+		if strings.Contains(ro.Group, "operator.openshift.io") {
+			relObj = append(relObj, ro)
+		}
+	}
+	if len(relObj) == 0 {
+		return nil
+	}
+	var res []clusterOperatorResource
+	for _, ro := range relObj {
+		key := fmt.Sprintf("%s-%s", ro.Group, strings.ToLower(ro.Resource))
+		versions, _ := resVer[key]
+		for _, v := range versions {
+			gvr := schema.GroupVersionResource{Group: ro.Group, Version: v, Resource: strings.ToLower(ro.Resource)}
+			clusterResource, err := i.dynamicClient.Resource(gvr).Get(i.ctx, ro.Name, metav1.GetOptions{})
+			if err != nil {
+				klog.V(2).Infof("Unable to list %s resource due to: %s", gvr, err)
+			}
+			if clusterResource == nil {
+				return nil
+			}
+			var ms, kind, name, apiVersion string
+			err = failEarly(
+				func() error { return parseJSONQuery(clusterResource.Object, "spec.managementState?", &ms) },
+				func() error { return parseJSONQuery(clusterResource.Object, "kind", &kind) },
+				func() error { return parseJSONQuery(clusterResource.Object, "apiVersion", &apiVersion) },
+				func() error { return parseJSONQuery(clusterResource.Object, "metadata.name", &name) },
+			)
+			if err != nil {
+				return nil
+			}
+			res = append(res, clusterOperatorResource{ManagementState: ms, Kind: kind, Name: name, APIVersion: apiVersion})
+		}
+	}
+	return res
+}
+
+func getOperatorResourcesVersions(i *Gatherer) (map[string][]string, error) {
+	resources, err := i.discoveryClient.ServerPreferredResources()
+	if err != nil {
+		return nil, err
+	}
+	resourceVersionMap := make(map[string][]string)
+	for _, v := range resources {
+		if strings.Contains(v.GroupVersion, "operator.openshift.io") {
+			gvA := strings.Split(v.GroupVersion, "/")
```

Comment:
What about to use `ParseGroupVersion` ?

Reply:
How can I use that? Where is it available please?

Reply:
It is in `schema.ParseGroupVersion` or here https://github.com/kubernetes/apimachinery/blob/master/pkg/runtime/schema/group_version.go#L211

Reply:
Thanks